### ISSUE TITLE
I stock in login page when I opened Udeler

### ISSUE THREAD

**Issue by noamangg** [rodrigosilvaluz/UDELER_ELECTRON#1]
I stock in login page when I use it when I press login it opens a new window then I login into my Udemy account put nothing happened
it opens like this
![Screenshot (171)](https://user-images.githubusercontent.com/71488276/148113688-36d20f9f-ff80-413a-8ab8-db1c19003b8e.png)

**Comment by Humam-Hamdan** [comment]
yes, and even if i use the token or the authen it never works.. the token just never work, but the auth chrome disables it
